pkg/gatherers/clusterconfig/container_images.go

```diff
@@ -17,28 +17,30 @@ import (
 	"github.com/openshift/library-go/pkg/image/reference"
 )
 
-const (
-	// imageGatherPodLimit is the maximum number of pods that
-	// will be listed in a single request to reduce memory usage.
-	imageGatherPodLimit = 200
-
-	// containerImageLimit is the maximum number of container images to collect.
-	// On average, information about one image takes up roughly 100 raw bytes.
-	containerImageLimit = 1000
-
-	// yyyyMmDateFormat is the date format used to get a YYYY-MM string.
-	yyyyMmDateFormat = "2006-01"
-)
-
-// GatherContainerImages collects essential information about running containers.
-// Specifically, the age of pods, the set of running images and the container names are collected.
+// GatherContainerImages Collects essential information about running containers. Specifically, the age of pods,
+// the set of running images and the container names are collected.
+//
+// ### API Reference
+// None
 //
-// * Location in archive: config/running_containers.json
-// * Id in config: clusterconfig/container_images
-// * Since versions:
-//   - 4.5.33+
-//   - 4.6.16+
-//   - 4.7+
+// ### Sample data
+// - docs/insights-archive-sample/config/running_containers.json
+//
+// ### Location in archive
+// - `config/running_containers.json`
+//
+// ### Config ID
+// `clusterconfig/container_images`
+//
+// ### Released version
+// - 4.7.0
+//
+// ### Backported versions
+// - 4.5.33+
+// - 4.6.1+
```

Comment:
This should be `4.6.16+` version, right? When looking at the original versions.

Reply:
4.5.33: [RN](https://access.redhat.com/errata/RHSA-2021:0428) [BZ](https://bugzilla.redhat.com/show_bug.cgi?id=1914255)
4.6.1: [RN](https://access.redhat.com/errata/RHBA-2020:4196) [BZ](https://bugzilla.redhat.com/show_bug.cgi?id=1860032)

Reply:
OK thanks....this one is confusing though